charts/reform-scan-blob-router/values.yaml

```diff
@@ -7,9 +7,6 @@ java:
       resourceGroup: reform-scan
       secrets:
         - app-insights-instrumentation-key
-        - error-notifications-password
-        - error-notifications-url
-        - error-notifications-username
```

Comment:
Can we also remove terraform config for copying secrets ?

Reply:
i thought about it but then feared and i think my fears are right (happened for us before) - secrets will be removed. will have to quickly prep a similar pr for notification service then. if we're happy - can address that separately with both prs ready to go in and master builds are not broken/flaky